### ISSUE TITLE
Layer toggling - layers in layer group don't show

### ISSUE THREAD

**Issue by roel-v** [Pattern-Projector/pattern-projector#50]
I'm using a PDF with layers that I found online as I don't have any sewing patterns with layers at hand: https://www.edcfiresafe.org/wp-content/uploads/2015/12/Using-Layerd-PDFs-with-Sample-Map.pdf

This PDF has its layers structured in what I'll call 'layer groups' (don't know the term in PDF parlance). All the layers show up in the PP layer list (including the 'nested' ones inside groups). But when I toggle those, they don't show. The only layer that does show/hide correctly is the one that is not inside a group, hence my hunch it has something to do with that, but I can't say for sure.

**Comment by roel-v** [comment]
This was tested on https://pattern-projector-git-layers-courtney-pattisons-projects.vercel.app/en/calibrate as of 10 mins ago BTW. Don't think I can see a git commit hash on that page?

**Comment by courtneypattison** [comment]
Yeah, I’m not sure about checking the version on the preview. I’ll try to fix the nested layers before deploying.

**Comment by courtneypattison** [comment]
Someone did report this issue a week or two ago. I think it was a love notions pattern.

**Comment by courtneypattison** [comment]
I'm finally looking into this issue and I think it was fixed at some point in the last couple of years. Probably by @cfcurtis ! I'm going to close it as fixed. I'm not sure if it's worthwhile showing the nesting structure since it's not common. There may be some cases where the naming is not sufficient to differentiate between layers that are nested, but it's so uncommon I'm not sure it's worth the trouble. 

<img width="1440" height="777" alt="Image" src="https://github.com/user-attachments/assets/8eda5f6f-1247-43b5-94b2-36e09198a367" />

<img width="1440" height="777" alt="Image" src="https://github.com/user-attachments/assets/efa4ee54-3650-4bb6-9fd1-25b84cc4f23b" />

**Comment by cfcurtis** [comment]
I don't think I can take credit for this one! I'm going to guess it's an upstream fix to the PDF library :)
